google-cloud-logging/src/test/java/com/google/cloud/logging/LoggingImplTest.java

```diff
@@ -1777,6 +1777,50 @@ public void testDeleteLogAsync() throws ExecutionException, InterruptedException
     assertTrue(logging.deleteLogAsync(LOG_NAME).get());
   }
 
+  @Test
+  public void testDeleteLogBillingDestination() throws ExecutionException, InterruptedException {
```

Comment:
nit: there is a possible inconsistency how tests validate different destinations in different interfaces. there are tests which do everything in one method: validating folder, project, org and billing acc destinations and like this change do it separately. it would be great to decide on one approach and follow it in all our tests.

Reply:
I prefer to reuse code even in tests